CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 
 ## vN.N.N
-* issue #1035: Add Spotify Embed Block
```

Comment:
@jalopez13 dont remove this line entirely, just remove the invalid issue number

Reply:
@efjacobson 

Removing issue number from description.